Helper/OmiseHelper.php

```diff
@@ -82,9 +83,104 @@ class OmiseHelper extends AbstractHelper
      */
     private $cardPaymentMethods = [
         Config::CODE,
+        Cc::CODE,
```

Comment:
The `Config::CODE` above is actually from `Cc`. I'm not sure why it's imported as `Config` either 😅 
```php
use Omise\Payment\Model\Config\Cc as Config;
```

Reply:
Ah yeah good point didn't see it import as Config Nice catch!! 🙏 
Fixed here -> https://github.com/omise/omise-magento/pull/361/commits/031312cabc07f229f17071daafa0ac046b977d2c